### PR TITLE
Update BUILD.md

### PR DESCRIPTION
**Pull request by priyadarshitathagat**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -48,3 +48,17 @@ pip3 install flask flask_swagger_ui paramiko flask-swagger-ui scp
...
+* Set the following flag in the /etc/docker/daemon.json file on the client:
+```
+{
+    "insecure-registries": ["registry-ip:port"]
+}
+```
+* Restart Docker
+```
+$ sudo systemctl restart docker
+```
```
Comment (aravindhank11) [inline BUILD.md:64]: Can we move this lines to https://github.com/vmware/te-ns/pull/21/files#diff-40f60e1037245d7b8a98a7325d53890a717da9979adeb54a61a795c4ba07f9c9R27 ?

nit: sudo --> [sudo]

Is registry-ip:port variables here ? Can registry-ip be 0.0.0.0 ? If so can we make it:
`0.0.0.0:$DOCKER_REGISTRY_PORT ?`, given that DOCKER_REGISTRY_PORT is export as:
```
export DOCKER_REGISTRY_PORT=6666
```

Comment (priyadarshitathagat) [reply BUILD.md:64]: This needs to be done on the client side, the registry-ip being the server, where we create the docker registry.

Comment (aravindhank11) [reply BUILD.md:64]: I thought you had hit issue when you were trying to run:
```
docker run -d -p $DOCKER_REGISTRY_PORT:$DOCKER_REGISTRY_PORT --restart=always --name registry -v /data/docker:/var/lib/registry -e REGISTRY_HTTP_ADDR=0.0.0.0:$DOCKER_REGISTRY_PORT registry:2
```

If it is common to both server and client -- can we add a new topic as "Known Issue and Fixes" and let us add the error thrown along with this fix. Would be helpful if registry-ip and port are made as variables

Comment (priyadarshitathagat) [reply BUILD.md:64]: This needs to be done only on the client, from where we will be pulling the image from the registry.

**Pull request closed without merging**